gui/functionality/adjust_timeouts.py

```diff
@@ -35,6 +45,10 @@ def __init__(self, data_provider, parent=None):
         self.data_provider.remove_cluster_signal.connect(self.handle_cluster_list_modification)
 
     def refresh_timeouts(self, cluster_name):
+        """ Fetches and refershes the timeouts of particular cluster.
```

Comment:
Typo: _refreshes_
A particular? Not sure

Reply:
fixed